config/katello-answers.yaml

```diff
@@ -42,9 +42,10 @@ puppet:
   server_foreman_ssl_key: /etc/pki/katello/puppet/puppet_client.key
 foreman_proxy:
   custom_repo: true
-  http: false
+  http: true
+  http_port: '8000'
```

Comment:
The default is an integer ```8000```. I'd expect this to fail but maybe hiera converts it to an integer. Either way, this can be removed.